### ISSUE TITLE
ContextKey not declared by package jwtmiddleware

### ISSUE THREAD

**Issue by kermit-sad-frog** [auth0/go-jwt-middleware#124]
### Describe the problem
ContextKey not declared by package jwtmiddleware

<img width="764" alt="Screenshot 2021-12-01 at 11 51 23" src="https://user-images.githubusercontent.com/89095027/144212473-43489bc7-aff6-4525-a976-e6bf090a7170.png">

### What was the expected behavior?

it should works how it was discribed in documentation 

### Reproduction

```
go get github.com/auth0/go-jwt-middleware
```
I
### Environment

```
require (
	github.com/auth0/go-jwt-middleware v1.0.1 // indirect
	github.com/aws/aws-sdk-go v1.38.0
	github.com/brianvoe/gofakeit/v6 v6.8.0
	github.com/gin-gonic/gin v1.7.4
	github.com/go-openapi/jsonreference v0.19.6 // indirect
	github.com/go-openapi/swag v0.19.15 // indirect
	github.com/go-resty/resty/v2 v2.6.0
	github.com/gofrs/uuid v4.0.0+incompatible
	github.com/google/go-querystring v1.1.0
	github.com/gotidy/fhir-client v0.2.3
	github.com/guregu/dynamo v1.11.0
	github.com/jarcoal/httpmock v1.0.8
	github.com/json-iterator/go v1.1.10 // indirect
	github.com/mailru/easyjson v0.7.7 // indirect
	github.com/swaggo/files v0.0.0-20210815190702-a29dd2bc99b2
	github.com/swaggo/gin-swagger v1.3.1
	github.com/swaggo/swag v1.7.3
	github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673
	golang.org/x/net v0.0.0-20210825183410-e898025ed96a
	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f
	golang.org/x/sys v0.0.0-20210831042530-f4d43177bf5e // indirect
	golang.org/x/text v0.3.7 // indirect
	gopkg.in/yaml.v2 v2.4.0
	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
)
```

**Comment by sergiughf** [comment]
Hey @kermit-sad-frog thanks for reporting this. We're in the process of cutting the new v2 release as right now when you do a `go get github.com/auth0/go-jwt-middleware` you'll still get served v1. https://github.com/auth0/go-jwt-middleware/tree/v1.0.1 

To try out the still beta version you could fetch directly on the latest commit with `go get github.com/auth0/go-jwt-middleware@9a48f965b89c5b31ad947abb7df6a4fff71a0453`. 

Apologies for the confusion! We're gonna make this a lot clearer soon.

**Comment by kermit-sad-frog** [comment]
thank you a lot for your answer, how I can handle this issue?
```
could not import github.com/auth0/go-jwt-middleware/validate/josev2 (no required module provides package "github.com/auth0/go-jwt-middleware/validate/josev2")
```

**Comment by sergiughf** [comment]
If you're using the beta version with `go get github.com/auth0/go-jwt-middleware@9a48f965b89c5b31ad947abb7df6a4fff71a0453` then you won't be using that package. I'd suggest clearing out your vendor folder `rm -rf vendor` and running `go mod vendor` again and checking your imports.

If instead you're using v1 through `go get github.com/auth0/go-jwt-middleware` you'll be needing that package and again a quick `rm -rf vendor` and `go mod vendor` should fix it. 

Let me know if you're still having issues:)

**Comment by jamra** [comment]
When do you think you're going to release your 2.0 version? 

I'm getting errors all over this library. Can you provide examples for beta branch? The example on the readme doesn't seem changed. Is my error the same?

github.com/auth0/go-jwt-middleware/validator: module github.com/auth0/go-jwt-middleware@latest found (v1.0.1), but does not contain package github.com/auth0/go-jwt-middleware/validator

**Comment by sergiughf** [comment]
Hey @jamra, we'll be cutting the new release tomorrow 8th of December so stay tuned! 

Apologies for the inconvenience so far, as soon as the release is out you'll be able to use the beta by running `go get github.com/auth0/go-jwt-middleware@v2.0.0-beta` 

Really appreciate you sticking with us so far and for your patience!

**Comment by jamra** [comment]
I’m in no hurry. I’ll be glad to test it when it’s released on the 8th.

**Comment by sergiughf** [comment]
We just released the [v2.0.0-beta](https://github.com/auth0/go-jwt-middleware/releases/tag/v2.0.0-beta) 🥳 ! 

You can start testing it by running `go get github.com/auth0/go-jwt-middleware/v2@v2.0.0-beta`. 

In case of issues fetching the v2 you might want to try `go clean --modcache` first before doing `go get`.

I'm closing this issue, but feel free to reopen if needed.

**Comment by jamra** [comment]
Okay so the example works and I am successfully able to validate the access token. I just can't find any examples on how to get the sub (and all the other token data). Can you please update the sample or show it here?
